README.md

```diff
@@ -245,6 +247,12 @@ If your desired ghc has been found, you use it to install haskell-language-serve
 ./cabal-hls-install hls-8.6.5
 ```
 
+### Installation from Hackage
+
+Direct installation from Hackage is supported via `cabal install haskell-language-server`.
```

Comment:
you could use `stack` to install it too

Reply:
I don't think so, since it's not in Stackage and therefore would require a ton of extra-reps.

Reply:
mmm you are right, it is possible with one command (`stack install brittany-0.13.1.0 Cabal-3.0.2.0 clock-0.7.2 ... haskell-language-server`) but not straighforward